docs/_documentations/installoncloud.md

```diff
@@ -10,26 +10,146 @@ order: 20
+    - You can modify this file, but leave the `spec.server.cheWorkspaceClusterRole` field set to `eclipse-codewind` and the `spec.storage.preCreateSubPaths` field set to `true`.
+2. If you're installing on a Kubernetes platform other than OpenShift, determine your Ingress domain. If you're unsure of your Ingress domain, ask your cluster administrator.
+    - Set the `spec.server.ingressDomain` field in the Che .yaml to the Ingress domain.
+3. Install Che:
+    - On OpenShift 3.x run the following command: `chectl server:start --platform=openshift --installer=operator --che-operator-cr-yaml=<codewind-che.yaml file>`
+    - On Kubernetes run the following command: `chectl server:start --platform=k8s --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=<codewind-che.yaml file>`
 
-### Add additional rules to support Codewind ODO extension
-The ODO extension needs to add additional rules for accessing OpenShift resources:
-1. In your home directory, run the following command to clone the ODO extension repository:
-`git clone https://github.com/eclipse/codewind-odo-extension`
+## Enabling privileged and root containers to run
+Codewind is required to run as privileged and as root, because it builds container images. If your cluster is OpenShift 3.x, run the following commands, where `<che namespace>` is the namespace you installed Che in:
+1. Enter `oc adm policy add-scc-to-group privileged system:serviceaccounts:<che namespace>` to enable privileged containers.
+2. Enter `oc adm policy add-scc-to-group anyuid system:serviceaccounts:<che namespace>` to enable containers to run as root.
+
+## After installing Che
+
+### Adding registries in Che
+After Che is started and running, add the container registry that will be used with Codewind.
+- On OpenShift or other Kube platforms, you can push your images to any Docker registry, such as Dockerhub, Quay.io, Google Cloud Registry (GCR), and more.
+- On IBM Cloud Private, you can push your image to any Docker registry *except* the internal Docker registry
+
+You'll need the following information to complete the instructions to add the registries:
+  - Server: `<registry-to-push-images-to>`
+  - Username: `<Your username>`
+  - Password: `<Your password>`
+
+For information on adding deployment registries to Che, consult our [registry documentation](https://www.eclipse.org/codewind/dockerregistry.html)
+
+## Creating the Codewind workspace
+
+### Confirm the Docker registry secrets.
+Confirm that you added the docker registry secrets in the Che dashboard. Go to **Administration**>**Add Registry** to check for the secrets.
+
+### Creating the Codewind workspace with a Devfile
+The general format for creating a Che workspace via a factory is:
+```
+http://<che ingress domain>/f?url=<hosted devfile URL>
+```
+
+We provide a ready-to-use devfile with the Codewind plug-ins. Enter the following URL to create a workspace from the devfile:
```

Comment:
Could we rewrite to..."Codewind includes a ready-to-use devfile with its plug-ins."?